src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  margin: 0;
+  width: min-content;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    list-style: none;
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
You need to create a universal variant for 1,2,3,4,5 starsline. 
Try it. Thanks)